app/src/main/java/de/westnordost/streetcomplete/screens/settings/quest_selection/QuestSelectionViewModel.kt

```diff
@@ -46,6 +59,11 @@ class QuestSelectionViewModelImpl(
     prefs: Preferences,
 ) : QuestSelectionViewModel() {
 
+    private val _searchText = MutableStateFlow(TextFieldValue())
+    override val searchText: StateFlow<TextFieldValue> = _searchText
```

Comment:
```suggestion
    override val searchText = MutableStateFlow(TextFieldValue())
```
(etc.)

...would be more consistent with the rest of the codebase. I.e. the implementation uses MutableStateFlow while the interface just exposes a StateFlow. So, I don't need to effectively duplicate all StateFlow fields (one private MutableStateFlow, one public StateFlow).

Reply:
Well, I am not desperate. I don't cast :-)

Reply:
Also, "desperate" is not the word I'd use for devs that break the interface by casting (instead of doing it properly). ;-)

Reply:
I meant in big projects sometimes managers push devs to make quick fixes. In those cases developers forget about clean code and start making mess in a project. Backing fields, Api/Impl and other patterns/techniques are meant to fight with that.

Reply:
Yeah, I know what you mean. ...And it is only a matter of time until such botch comes back haunting the developers, oftentimes not even hitting the dabblers but those that come after... 😬 Hrr, memories, that gives me the creeps. 

(So, maybe the pattern you use is generally preferable, as a guard against such practices. But well, being kind of the BDFL in this project, I won't allow botch in the first place, so such safeguard is less necessary. Readability and consistency is higher on my list.)